app/views/publishers/vacancies/build/applying_for_the_job.html.slim

```diff
@@ -13,7 +13,18 @@
 
         h2.govuk-heading-l = t("publishers.vacancies.steps.applying_for_the_job")
 
-        - if vacancy.listed? || current_organisation.local_authority?
+        - if vacancy.job_roles.include?("education_support")
```

Comment:
If you rebase, you should be able to do `vacancy.primary_job_role == "education_support"` instead